src/services/sentry.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+import * as Sentry from '@sentry/react-native'
+
+export const initSentry = (): void => {
+  if (!__DEV__) {
+    Sentry.init({
+      dsn: 'https://8e99e20eb419461d8a539a15e09199d8@sentry.tuerantuer.org/3'
+    })
+  }
+}
+
+export const log = (message: string, level = 'debug'): void => {
+  if (__DEV__) {
+    switch (level) {
+      case Sentry.Severity.Fatal:
+      case Sentry.Severity.Critical:
+      case Sentry.Severity.Error:
+        console.error(message)
+        break
+      case Sentry.Severity.Warning:
+        console.warn(message)
+        break
+      case Sentry.Severity.Log:
+        console.log(message)
+        break
+      case Sentry.Severity.Info:
+        console.info(message)
+        break
+      case Sentry.Severity.Debug:
+        console.debug(message)
+        break
+    }
```

Comment:
Is there no function from sentry which accepts a log level?

Also shouldn't the `level` variable have a proper type? I would suspect something like 'debug' | 'error'

Reply:
Copied from integreat which was implemented by me. I struggled with the correct typing. But there probably really is a better way to do this.

Reply:
As it is working/fine for Integreat and this is just the setup i will keep it, if there is a need to change i can do in a separate ticket.